addons/onboarding/models/__init__.py

```diff
@@ -5,3 +5,4 @@
 from . import onboarding_onboarding_step
 from . import onboarding_progress
 from . import onboarding_progress_step
+from . import company
```

Comment:
small detail: since it's currently alphabetical we could keep the order (and while we are at it, it's better to call  it res_company.py, there is 6 cases where the file is called company.py and 168 where it's called res_company.py)